.travis.yml

```diff
@@ -11,11 +11,11 @@ script:
   - set -o pipefail
   - xcodebuild test -scheme Tyro -configuration Debug | xcpretty -c
   - xcodebuild clean -scheme Tyro -configuration Debug
-  - xcodebuild test -scheme Tyro-iOS -configuration Debug -sdk iphonesimulator -destination "platform=iOS Simulator,name=iPhone 6s" | xcpretty -c
+  - xcodebuild test -scheme Tyro-iOS -configuration Debug -sdk iphonesimulator -destination "platform=iOS Simulator,name=iPhone 8" -UseModernBuildSystem=NO | xcpretty -c
```

Comment:
why did we need this UseModernBuildSystem=NO ? isn't better to use the new build system?

Reply:
If I run the command with this it gives me the below error "Multiple targets match implicit dependency for product reference 'Swiftz.framework'. Consider adding an explicit dependency on the intended target to resolve this ambiguity. (in target 'Tyro-iOSTests' from project 'Tyro')"  Because of this we are using this attribute in command